### DISCUSSION TITLE
Is there an equivalent docker image for whitesource/renovate ?

### DISCUSSION THREAD

**Discussion by MadaraUchiha-314** [mend/renovate-ce-ee#437] [Q&A] [answered]
The latest version of the docker image [whitesource/renovate](https://hub.docker.com/layers/whitesource/renovate/5.1.0/images/sha256-dd0ce73652a3cb2cde549bf0d7deddda4a1604a7438b537d24d1e418bfa50a88?context=explore) has a lot of security issues due to some of the packages installed.

I see that no new version of it has been published since 4 months. Is there an new docker image for the same ?

I see that there are: 
- [renovate-ce](https://github.com/orgs/mend/packages/container/package/renovate-ce)
- [renovate-ee-worker](https://github.com/orgs/mend/packages/container/package/renovate-ee-worker)
- [renovate-ee-server](https://github.com/orgs/mend/packages/container/package/renovate-ee-server)

Are any of these suitable replacements for the one I was using ?

**Comment by rarkins** [comment, marked as answer]
Yes, Renovate CE. See https://github.com/mend/renovate-ce-ee/releases/tag/6.0.0
